use_cases_and_horizontal_approaches/insurance_pricing/Code first Incurred Claims - Insurance pricing.ipynb

```diff
@@ -159,7 +159,9 @@
     "df[\"EndDate\"] = pd.to_datetime(df[\"EndDate\"])\n",
     "\n",
     "# 2. Calculate the 'Exposure' feature based on the policy's duration in years, considering the start and end date.\n",
-    "df[\"Exposure\"] = ((df[\"EndDate\"] - df[\"EffectiveDate\"]) / np.timedelta64(1, \"D\") + 1) / 365\n",
+    "df[\"Exposure\"] = (\n",
+    "    (df[\"EndDate\"] - df[\"EffectiveDate\"]) / np.timedelta64(1, \"D\") + 1\n",
+    ") / 365\n",
```

Comment:
black seemed to not like these, so I went ahead and updated it as well.

Reply:
Thanks